src/main/resources/application.properties

```diff
@@ -57,3 +57,8 @@ gingersnap.region.us-east.backend.key-columns=fullname
 
 #quarkus.log.category."io.debezium.connector.sqlserver".level=TRACE
 #quarkus.log.category."io.debezium.connector.sqlserver".min-level=TRACE
+
+## Kubernetes Configuration
+gingersnap.k8s.enabled=false
```

Comment:
I am thinking what if just the presence of the name means it is enabled? Seems a bit superfluous. And then we also won't need the extra inner Configuration class above.

Reply:
Updated.